torch/_overrides.py

```diff
@@ -111,12 +111,13 @@ def handle_torch_function(
     """
     # Check for __torch_function__ methods.
     overloaded_args = _get_overloaded_args(relevant_args)
+    types = tuple(map(type, overloaded_args))
```

Comment:
I think `_get_overloaded_args` guarantees that the entries in `overloaded_args` all have unique types, but it may be worth adding a comment to that effect or doing `tuple(set(map(type, overloaded_args)))` instead.